Delivery-Service.API/Delivery-Service.API.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RootNamespace>Delivery_Service.API</RootNamespace>
+    <OutputType>Exe</OutputType>
```

Comment:
switch to class lib project type

Reply:
if i set Delivery_Sevice.API as class lib, i have error "unble to run your project. the RunCommand property is not defined"

Reply:
ok